vercel.json

```
@@ -6,5 +6,8 @@
       "source": "/api/prereg",
       "headers": [{ "key": "Access-Control-Allow-Origin", "value": "*" }]
     }
+  ],
+  "redirects": [
```

You’ll want to use `rewrites` here so the routes don’t redirect
```suggestion
  "rewrites": [
```